src/tool/subcommands/state_migration_cmd.rs

```diff
@@ -6,8 +6,8 @@ use crate::utils::db::car_stream::{CarStream, CarWriter};
 use crate::utils::net::global_http_client;
 use anyhow::ensure;
 use async_compression::tokio::write::ZstdEncoder;
-use futures::{stream, StreamExt as _, TryStreamExt as _};
-use itertools::Itertools as _;
+use futures::{stream, StreamExt, TryStreamExt};
+use itertools::Itertools;
```

Comment:
You don't like the `as _` pattern?

Reply:
I do, it get somewhat pruned during merge.